contrib/test_jenkins.sh

```diff
@@ -817,38 +817,45 @@ test_malloc_hooks_mpi() {
 # Run tests with MPI library
 #
 run_mpi_tests() {
-	if module_load hpcx-gcc && mpirun --version
+	mpi_module=hpcx-ga-gcc
+	if module_load ${mpi_module}
 	then
-		# Prevent our tests from using UCX libraries from hpcx module by prepending
-		# our local library path first
-		save_LD_LIBRARY_PATH=${LD_LIBRARY_PATH}
-		export LD_LIBRARY_PATH=${ucx_inst}/lib:${LD_LIBRARY_PATH}
+		if mpirun --version
+		then
+			# Prevent our tests from using UCX libraries from hpcx module by prepending
+			# our local library path first
+			save_LD_LIBRARY_PATH=${LD_LIBRARY_PATH}
+			export LD_LIBRARY_PATH=${ucx_inst}/lib:${MPI_HOME}/lib
```

Comment:
why previous LD_LIBRARY_PATH is removed?

Reply:
right, need to rework it